### PR TITLE
Feature Explorer and Recommender Docs

### PR DESCRIPTION
**Pull request by dattranm**
Add Feature Explorer and Recommender Docs:
- Create new folder, feature_recommender, inside anovos-modules-overview and push docs in index.md
- Add Feature Explorer and Recommender to Overview section (overview/index.md)
- Add Feature Explorer and Recommender to mkdocs.yml to serve the following section.

### CODE REVIEW

File: docs/anovos-modules-overview/overview/index.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ The key modules included in the alpha release are:
     In the alpha release, the data transformer module only includes some basic pre-processing functions like binning, encoding, to name a few. These functions were required to support computations of the above key modules.  A more exhaustive set of transformations can be expected in future releases.
 5. **Data Report**:
     This module is a visualization component of Anovos. All the analysis on the key modules is visualized via an HTML report to get a well-rounded understanding of the ingested dataset. The report contains an executive summary, wiki for data dictionary & metric dictionary, a tab corresponding to key modules demonstrating the output.
+6. **Feature Explorer and Recommender**:
```
Comment (kajanansangar) [inline docs/anovos-modules-overview/overview/index.md:25]: this should be moved up to number 4

File: docs/anovos-modules-overview/overview/index.md [hunk 2]
```diff
@@ -22,6 +22,14 @@ The key modules included in the alpha release are:
...
+    This module consists of methods and functions to help users figure out which features would help them in solving the given busines problem, which is a cold-start problem for almost all the data scientists. Feature Recommender has two sub-components in it, namely, Feature Explorer & Feature Recommender.
+       
+    a. *Feature Explorer*: 
+    This sub-module featuring a list of methods, allowing users to explore existing features based on their desired industry and/or use case.
+        
+    b. *Feature Recommender*: 
+    This sub-module featuring 3 methods, to recommend features to users based on their ingested data dictionary consists of attribute name & attribute description. Further this module provides a comprehensive mapping method from the ingested data dictionary to either available features, or their own feature corpus.
 
 Note: Upcoming Modules - Feature Wiki, Feature store, Auto ML, ML Flow Integration
```
Comment (kajanansangar) [inline docs/anovos-modules-overview/overview/index.md:34]: This should be changed to Feature Store Integration, Auto ML, & ML Flow Integration

**Review by kajanansangar — CHANGES_REQUESTED** [review]
Changes  are proposed

**Comment by ionicsolutions** [comment]
@dattranm For the most part, we will integrate this text into the API docs, i.e., we will store it in the main repository right with the source code.

Let me know when you're done here and I'll revise and move the text so that it's included in the documentation :-)

**Comment by kajanansangar** [comment]
@ionicsolutions respective changes here should be copied over to anovos/anovos since the anovos/anovos-doc is no more available

**Comment by ionicsolutions** [comment]
@kajanansangar Should I take care of that or will @dattranm add his work in the main repo?

**Pull request merged**